### ISSUE TITLE
Train loss

### ISSUE THREAD

**Issue by zhaoyj96** [czq142857/BSP-NET-pytorch#19]
Hi,

Thank you for the code. I am using my own data to train the AE model and meet some problems. When I train with size 16, train loss is normal, probably between 0.001 and 0.002. But when I train with size 32, the loss is about 0.02. When I train with size 64, the loss is about 0.05. I want to ask whether the loss is normal. The batch size is 24 for all training.

**Comment by czq142857** [comment]
I am not sure whether the range of the loss is normal or not. But the loss will be higher when training on a higher resolution. So in your case, it is normal for the loss to jump from 0.002 to 0.02 to 0.05 when the training resolution switches from 16 to 32 to 64.

To confirm whether the training is successful, check the quality of the sample output shapes during training.

**Comment by zhaoyj96** [comment]
Thank you for your reply. I found some problems with my ground truth data.
![image](https://user-images.githubusercontent.com/67319838/233821922-7941b2e0-fa9f-455a-bf4e-40d4eeed9951.png)
Its interior is hollow. I think some problems happened in data processing using [https://github.com/czq142857/IM-NET/tree/master/point_sampling](url)

**Comment by czq142857** [comment]
It is likely due to improper normalization. Please normalize your shapes (voxels) so that the object is centered and the diagonal of the object bounding box is 1.

**Comment by zhaoyj96** [comment]
Thank you for your reply. I'm sorry I don't know how to normalize. Is it using the binvox?

**Comment by czq142857** [comment]
You can refer to *1_simplify_obj.py* and *2_voxelize.py* in [this data preparation code](https://github.com/czq142857/DECOR-GAN/tree/main/data_preparation) for normalizing shapes in obj formats and voxelizing them, respectively.
